go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/mittwald/vaultgo
+module github.com/vejed/vaultgo
```

Comment:
```suggestion
module github.com/mittwald/vaultgo
```

Reply:
I chooe invalid source branch, sorry.
Please, close this PR and view another one (other conversations are also fixed) https://github.com/mittwald/vaultgo/pull/21